lib/bundler/source/path/installer.rb

```diff
@@ -11,8 +11,8 @@ def initialize(spec, options = {})
           @gem_bin_dir       = "#{Bundler.rubygems.gem_dir}/bin"
           @bin_dir           = Bundler.requires_sudo? ? @tmp_bin_dir : @gem_bin_dir
           @gem_dir           = Bundler.rubygems.path(spec.full_gem_path)
-          @wrappers          = options[:wrappers] || true
-          @env_shebang       = options[:env_shebang] || true
+          @wrappers          = true
```

Comment:
I'd rather not start allowing it to be set if we haven't allowed it this entire time so far....

On Mon, May 4, 2015 at 6:38 PM, Samuel E. Giddins
<notifications@github.com> wrote:

>> @@ -11,8 +11,8 @@ def initialize(spec, options = {})
>>            @gem_bin_dir       = "#{Bundler.rubygems.gem_dir}/bin"
>>            @bin_dir           = Bundler.requires_sudo? ? @tmp_bin_dir : @gem_bin_dir
>>            @gem_dir           = Bundler.rubygems.path(spec.full_gem_path)
>> -          @wrappers          = options[:wrappers] || true
>> -          @env_shebang       = options[:env_shebang] || true
>> +          @wrappers          = true
> maybe this should be `options.fetch(:wrappers, true)`?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/bundler/bundler/pull/3616/files#r29639504